content/chapter/02.tex

```diff
@@ -10,7 +10,7 @@ \section{Mikrocontrollerplattform Arduino}
 Beispielhaft ist dies in Abbildung \ref{fig:arduino_uno_schema} anhand eines Arduino UNOs dargestellt.
 \begin{figure}[H]
 	\centering
-	\includegraphics[width=5cm]{images/chapter/02/arduino_uno.png}
+	\includegraphics[width=12cm]{images/chapter/02/arduino_uno.png}
```

Comment:
Die Shields aber schon in 12cm, interpretiere ich das richtig, ja?
Denn mit 8cm wären die Shields schon ein wenig klein.